src/kem/bike/CMakeLists.txt

```diff
@@ -60,7 +60,16 @@ if(OQS_ENABLE_KEM_bike_l1 OR OQS_ENABLE_KEM_bike_l3)
         set_source_files_properties(${AVX512_SRCS} PROPERTIES COMPILE_OPTIONS "${AVX512_FLAGS}")
 
         set_source_files_properties(additional_r3/gf2x_mul_base_pclmul.c PROPERTIES COMPILE_OPTIONS "-mpclmul;-msse2")
-        set_source_files_properties(additional_r3/gf2x_mul_base_vpclmul.c PROPERTIES COMPILE_OPTIONS "-mvpclmulqdq;${AVX512_FLAGS}")
+
+        # Some older compilers don't support the "mvpclmulqdq" flag so we have to check.
+        try_compile(VPCLMUL_SUPPORTED ${CMAKE_BINARY_DIR}
```

Comment:
You could also modify `.CMake/detect_gcc_clang_intrinsics.c` to ensure we set an appropriate OQS_USE_XXX_INSTRUCTIONS flag in  `gcc_clang_intrinsics.cmake`.

Reply:
Please make this change for generality.

Reply:
done